### PR TITLE
fix(tem): bimi MTA-4668

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: managed-services/transactional-email/how-to/bimi.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- A logo of your brand saved in SVG format
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status of the Organization in which the actions will be performed, or be an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)
+
+## How does BIMI work?
+
+Before your brand's logo can be displayed, your email must first pass authentication checks. Checks are usually performed through [SPF (Sender Policy Framework)](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain), [DKIM (DomainKeys Identified Mail)](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain), and [DMARC (Domain-based Message Authentication, Reporting, and Conformance)](/managed-services/transactional-email/how-to/add-dmarc-record) checks, which are supported by Scaleway. Some email clients also require a Verified Mark Certificate (VMC) to ensure the logo's authenticity.
+
+BIMI is a DNS TXT record that you set up to indicate what brand's logo. The record contains the URL to the brand's logo in SVG (Scalable Vector Graphics) format.
```
Comment (nerda-codes) [inline managed-services/transactional-email/how-to/bimi.mdx:32]: I don't understand what you mean with the first sentence, I think something's missing.

File: managed-services/transactional-email/how-to/bimi.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+3. Click the **DNS zones** tab. A list of the DNS zones you have configured within the selected domain displays.
+4. Click **+ Add records** to add new records to your DNS zone. A pop-up displays.
+5. Fill in the required information for the record.
+
+    <Lightbox src="scaleway-bimi-record.webp" alt="" />
+
+    | Field | Value   | Notes |
+    | ------| ------- | ----- |
+    | Type  | TXT           |      |
+    | Name  | `default._bimi.<example.com>`   | At Scaleway your domain name is added automatically, so you need only add `default._bimi`. Other DNS clients might require you to add the entire address.  |
```
Comment (nerda-codes) [inline managed-services/transactional-email/how-to/bimi.mdx:51]:
```suggestion
    | Name  | `default._bimi.<example.com>`   | At Scaleway your domain name is added automatically, so you only need to add `default._bimi`. Other DNS clients might require you to add the entire address.  |
```

Comment (nerda-codes) [reply managed-services/transactional-email/how-to/bimi.mdx:51]: I know the original sentence is correct but as a user, I'd be confused to see it written like that, but ffti

File: menu/navigation.json [hunk 3]
```diff
@@ -3315,6 +3315,10 @@
                     "label": "Generate API keys for API and SMTP sending with IAM",
                     "slug": "generate-api-keys-for-tem-with-iam"
                   },
+                  {
+                    "label": "Set up a BIMI record",
+                    "slug": "bimi"
```
Comment (bene2k1) [inline menu/navigation.json:3320]:
```suggestion
                    "slug": "configure-bimi"
```
maybe add a verb in the slug like `configure` or `setup`

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**